admin/products_to_categories.php

```diff
@@ -20,7 +20,13 @@
 $_GET['products_filter'] = $products_filter;
 
 // current_category: the category the selected product is in
-$current_category_id = (int)(isset($_GET['current_category_id']) ? $_GET['current_category_id'] : $current_category_id);
+if (!empty($_POST['current_category_id'])) { // page load from change of selection in product listing form
+    $current_category_id = (int)$_POST['current_category_id'];
+} elseif (!empty($_GET['current_category_id'])) { // page load from a redirect
+    $current_category_id = (int)$_GET['current_category_id'];
+} else {
+    $current_category_id = 0;
+}
```

Comment:
In 1.5.8 since we require minimum PHP 7 this would work too:
```php
$current_category_id = (int)($_POST['current_category_id'] ?? $_GET['current_category_id'] ?? 0)
```

Reply:
YUCK!

Reply:
I just used similar to what was already there: changed them to use the php7 constructs.

Reply:
I'm referring to the use of ??.  Ugh.  I greatly prefer what you did.

Reply:
😁